docs/content/en/main/concepts/investigations/edge-multi-cluster copy.md

```diff
@@ -0,0 +1,135 @@
+ create workloads that are node-agnostic and can be constrained to operate in places where certain conditions are met. Operators enjoy
+  gaurantee that workloads are rescheduled to work wherever constraints allow them to run. This approach has satisfied the large
+   majority of use cases in the cloud native community.
+
+There is a growing community of cloud native developers and operators that work for organizations that require more fine-grain control
+ over the distribution of workload due to the nature of their business. Telecommunications, industrial manufacturers, and automotive
+  companies would like to rely on Kubernetes state-based management but they need more flexibility to accomodate for conditions like
+   large-scale deployments, disconnected and autonomous operation, and constrained cluster size. These requirements represent the Edge 
+   Multi-cluster use case.
+
+A key area of investigation for `kcp` is exploring large-scale deployment of workloads that can operate under constraints and
+ conditions unlike those accomodated by upstream Kubernetes. Ideally, we want declaritively defined workloads to operate across a
+  hierarchy combining KCP workspaces and Kubernetes cluster namespaces to achieve greater scale and control over placement. Deployment
+   of workloads across many thousands of destinations that include namespaced and non-namespaced objects, receiving state and reporting
+    status (grouped and individually), require new interfaces based on (and extending) KCP primitives.
+
+## Goal: Placement strategies
+
+There are many different ways to characterize deployments of workloads in the Edge use case. Support for blue/green, canary, carbon
+ intensity sensitive, and threat avoidance are some of the deployment strategies to consider when casting out 100's or 1000's of
+  workloads to millions of destinations. Having the ability to assign workloads using a declaritively defined placement strategy to
+   edge cluster locations, even if they are offline at times, is a key requirement of the edge multi-cluster use case.
+
+## Overlap of roles, responsibilities, integrations, and security
+
+Today's smartphone is not only the property of the user who purchased it. The reality is that application developers (app store),
+ corporate entities (vpn profiles), and the smartphone manufacturer have roles and responsibilities on these devices. Edge cluster
+  locations are no different in this respect. A scalable Edge multi-cluster solution can thrive and advance if there is support for
+   community contribution powered by integration and proper security features. All of these considerations must work across an evolving
+    hierarchy that combines small and large control plane API services.
+
+## Hierarchy and Scale are core considerations
+
+KCP has introduced the concept of transparent multi-cluster which can be used to expand the number of entry points for workloads and
```

Comment:
The first sentence is unclear to me.  What is an "entry point for workloads and operations"?  Is that "workloads and operations" trying to get at a noun-verb duality, or is a distinction between dev and ops?  What sort of "entry point" is meant?  Is it something like an endpoint for communication, or a way into thinking about a lot of stuff, or ... ?  What has "reducing the size of the control plane to a single binary" got to do with any of that?  Remember also that (a) the kcp binary is prepared to work with an external etcd cluster and this is the only sensible architecture for production use and (b) by the time we get sharding the picture is more complicated.

Reply:
I changed this paragraph to mention kcp's minimal api server and kcp's TMC as two separate concepts.